developer/testscripts/aatest_ds000114_fmrilocalpipeline.m

```diff
@@ -0,0 +1,121 @@
+function aatest_ds000114_fmrilocalpipeline(parameterfile, deleteprevious,wheretoprocess)
+
+% This script runs a local pipeline connection to first pre-process the study,
+% and then does modelling. It is the same script as ds000114_fmri except that
+% it uses a pipeline connection to use preprocessed results for modelling.
+% The script also uses symlinks to load stream files to the new modules instead of copying.
+
+% -------------------------------------------------------------------------
+% init
+% -------------------------------------------------------------------------
+
+aap = aa_test_inittest([mfilename('fullpath') 'prep'], parameterfile, deleteprevious, wheretoprocess);
+% -------------------------------------------------------------------------
+% First pre-process the data.
+% -------------------------------------------------------------------------
+
+aap.options.autoidentifystructural_choosefirst = 1;
+aap.options.autoidentifystructural_chooselast = 0;
+
+aap.options.remotesymlinks = 1; % Use symlinks instead of copying
+aap.options.NIFTI4D = 1;
+aap.acq_details.numdummies = 4;	
+aap.acq_details.numdummies = 1;
+aap.acq_details.input.correctEVfordummies = 1;
+
+aap.tasksettings.aamod_segment8.writenormimg = 0;
+aap.tasksettings.aamod_segment8.samp = 2;
+aap.tasksettings.aamod_slicetiming.autodetectSO = 1;
+aap.tasksettings.aamod_slicetiming.refslice = 16;
+aap.tasksettings.aamod_smooth.FWHM = 5;
+
+prep_dir = fullfile(aap.acq_details.root,[aap.directory_conventions.analysisid]);
+first_dir = pwd;
+
+% -------------------------------------------------------------------------
+% BIDS
+% -------------------------------------------------------------------------
+
+aap.acq_details.input.combinemultiple = 1;
+
+% five subjects is about minimum to run a second level model
+
+aap = aas_processBIDS(aap,[],{'finger_foot_lips','line_bisection'},{'sub-01','sub-02','sub-03'});
+
+aa_doprocessing(aap);
+
+aa_close(aap);
```

Comment:
The latest error message indicates that SPM is not loaded. I suggest removing this line because closing aa usually removes SPM from the path.

Reply:
I removed that bit, however it still crashes  for the same reason :(